integration/grpc-web-no-streaming-observable/example.ts

```diff
@@ -321,7 +321,11 @@ export const Empty = {
  * but with the streaming method removed.
  */
 export interface DashState {
-  UserSettings(request: DeepPartial<Empty>, metadata?: grpc.Metadata): Observable<DashUserSettingsState>;
+  UserSettings(
+    request: DeepPartial<Empty>,
+    abortSignal?: AbortSignal,
```

Comment:
@hersentino this seems fine, but it's probably a breaking API change to current codes that are passing `foo.UserSettings(request, metadata)`?

Can we push the `abortSignal` last so that it's not a breaking change?

Reply:
Yep no problem :). 

I initially put it in second argument because in this it was set as 2nd argument in this [PR](https://github.com/stephenh/ts-proto/pull/731/files#diff-c74bac0693e72660a78a574521c3dadd5dfa74fecb9d2dba513c7407897cb39bR65)

So I had to also push abortSignal to last argument for the function `generateService` inside `src/generate-services.ts`.  Not sur if this modification break something somewhere else.